### DISCUSSION TITLE
Firewall will not honour Metered/Unmetered

### DISCUSSION THREAD

**Discussion by iulko** [celzero/rethink-app#998] [General]
a

**Comment by ignoramous** [comment]
Can you see if this is still the case with [`v055a`](https://github.com/celzero/rethink-app/releases/tag/v0.5.5a) released a few days ago: ([website](https://rethinkdns.com/download), [playstore](https://play.google.com/store/apps/details?id=com.celzero.bravedns))?

If so, would you be comfortable sharing a screen recording of this bug showing an app that's setup to block on metered connection not really getting blocked (in the *Network Log*)?
